### DISCUSSION TITLE
New release v0.5.x

### DISCUSSION THREAD

**Discussion by roromix** [pdfcpu/pdfcpu#726] [General]
Is it possible to create a new release?
I use the api and I cannot import the master following the pull requests that you merged

`
/go/src/my-go-project # go get -u github.com/pdfcpu/pdfcpu@master
go: downloading github.com/pdfcpu/pdfcpu v0.5.1-0.20230926190146-4a2298ca0483
go: github.com/pdfcpu/pdfcpu@master: github.com/pdfcpu/pdfcpu@v0.5.1-0.20230926190146-4a2298ca0483: verifying module: github.com/pdfcpu/pdfcpu@v0.5.1-0.20230926190146-4a2298ca0483: reading https://sum.golang.org/lookup/github.com/pdfcpu/pdfcpu@v0.5.1-0.20230926190146-4a2298ca0483: 404 Not Found
        server response: not found:
`

**Comment by hhrutter** [comment]
I think #567 will help you with this.
Expect a new release around the end of the month.
